CRM/Contribute/Form/Contribution/Main.php

```diff
@@ -1204,7 +1204,7 @@ public function submit($params) {
       }
     }
 
-    $params['amount'] = $this->getMainContributionAmount();
+    $params['amount'] = $params['total_amount'];
```

Comment:
This seems to be triggering the test failures

Reply:
I think we don't want to change this - just change the places where it might be used - more specificity is better & this is very much not clear & specific